package.json

```diff
@@ -4,8 +4,9 @@
   "homepage": "https://github.com/jcode999/Mind_Reader/wiki",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/jcode999/Mind_Reader.git"
+    "url": "git+https://github.com/kelsvilla/LegoMindstormsVSExtension"
```

Comment:
If you're wondering, the URL is set to our repo so that the extension gets the most up-to-date changes